tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
 # To build with GPU support:
 $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
 
-$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
+$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
 
 # The name of the .whl file will depend on your platform.
-$ pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-linux_x86_64.whl
+$ sudo pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-any.whl
```

Comment:
>
>  # The name of the .whl file will depend on your platform.
> -$ pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-linux_x86_64.whl
> +$ sudo pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-any.whl

Sure it is depend on the platform, In case of OS X, the pip and python
installation need to sudo permission.
In this reason, using OS X enviroment, the pip install description on "
os_setup.md", almost add sudo command.
This commit is just add sudo command on OS X. I do same install from source
code on Ubuntu 14.04. it is not use sudo permission.


On Tue, Apr 26, 2016 at 8:17 AM, Martin Wicke <notifications@github.com>
wrote:

> In tensorflow/g3doc/get_started/os_setup.md
> <https://github.com/tensorflow/tensorflow/pull/2095#discussion_r61006354>:
>
> >
> >  # The name of the .whl file will depend on your platform.
> > -$ pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-linux_x86_64.whl
> > +$ sudo pip install /tmp/tensorflow_pkg/tensorflow-0.8.0-py2-none-any.whl
>
> We can make it more explicit that the name depends on the platform, maybe
> in the text below? We can also add a note there that depending on your pip
> and python installation, you may need to use sudo on this command. I'm a
> little wary about adding it for everybody, it really shouldn't be required.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2095/files/7d1417ce568bb5d28b215d7272549751cd599c63#r61006354>
>